MesnardBarba-CiSE.md

```diff
@@ -344,13 +345,13 @@ And report negative results!
 Understandably, long 3D simulations that take huge computational resources may not be feasible to repeat. 
 We should continue the conversation about what it means to do reproducible research in high-performance computing (HPC) scenarios. 
 When large simulations run on specific hardware with one-off compute allocations, they are unlikely to be reproduced. 
-In this case, it is even more important that the researchers advanced to the HPC application on a solid progression of fully reproducible research at the smaller scales. 
+In this case, it is even more important that researchers advance towards these HPC applications on a solid progression of fully reproducible research at the smaller scales. 
 
 Computational science and engineering makes ubiquitous use of linear algebra libraries like PETSc, Hypre, Trilinos and many others. 
 Rarely do we consider that using different libraries might produce different results. 
 But that is the case. 
 Sparse iterative solvers use various definitions of the _tolerance_ criterion to exit the iterations, for example. 
-The very definition of _residual_ could be different. 
+The very definition of _residual_ could be different. (Clarify: the _residual_ is the same, but the _relative residual_ can be calculated in various ways and the _exit criterion_ can vary widely)
```

Comment:
This comes from a comment by @piyueh ... and said specifically that the residual is not always defined the same.